teslajsonpy/car.py

```diff
@@ -537,6 +537,18 @@ def window_rp(self) -> int:
         """Return rear passenger window status."""
         return self._vehicle_data.get("vehicle_state", {}).get("rp_window")
 
+    @property
+    def is_window_open(self) -> bool:
```

Comment:
This is minor but the other properties are checking if things are closed, `is_frunk_closed`, `is_trunk_closed`, etc. Just to match everything else, maybe switch this to `is_window_closed` and update the returns to reflect the state.